lang/nl/invoice.php

```diff
@@ -3,7 +3,7 @@
 return [
     /*
     |--------------------------------------------------------------------------
-    | Norwegian Invoice Language Lines
```

Comment:
The language code for Norwegian is `no`. The language code for Dutch is `nl`

Reply:
probably here is better to rename norwegian into dutch instead of deleting line.

Reply:
With changing of the base the whole line got deleted, while I updated the line with the rename like you said